utils/update-catalog-hashes-cli.ts

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env -S node -r @swc-node/register
+/**
+ * CLI script to update catalog hashes in package.json files.
+ *
+ * This script updates the 'catalogHash' field in package.json files for published
+ * packages when their catalog dependencies have changed. This helps ensure that
+ * packages are rebuilt when their catalog dependencies are updated, even if the
+ * package itself hasn't changed.
+ */
+import process from "node:process";
+
+import {updateCatalogHashes} from "./update-catalog-hashes";
+
+function printHelp() {
+    console.log("Usage: update-catalog-hashes [options]");
+    console.log("");
+    console.log("Updates catalog hashes in package.json files");
+    console.log("");
+    console.log("Options:");
+    console.log(
+        "  --dry-run    Show what would be updated without making changes",
+    );
+    console.log(
+        "  --verbose    Show detailed information about catalog dependencies",
+    );
+    console.log("  --help       Show this help message");
+    console.log("");
+    console.log("Examples:");
+    console.log("  update-catalog-hashes");
+    console.log("  update-catalog-hashes --dry-run");
+    console.log("  update-catalog-hashes --verbose");
```

Comment:
These aren't useful unless they show more complex usages of flags.  As it stands, there's only two flags and they're pretty common, so examples here seems like just extra noise.